menas/ui/components/login/loginDetail.controller.js

```diff
@@ -94,6 +94,7 @@ sap.ui.define([
       if (oControl) oControl.setBusy(true);
 
       let fnSuccess = (result, status, xhr) => {
+        this.byId("password").setValue("");
```

Comment:
Seems like it does not work. And I cannot find any ID password in the DOM. Is `byId` grabbing by partial ID?

Reply:
Make sure to test with a user/password which isn't in your autocomplete.

Reply:
P.S its a ui5 element, you won't find it in the dom by this id.

Reply:
> Make sure to test with a user/password which isn't in your autocomplete.

I am testing in a private window with tor. Works fine now.